diff --git a/third_party/blink/renderer/platform/graphics/canvas_resource_provider.cc b/third_party/blink/renderer/platform/graphics/canvas_resource_provider.cc
index 5ae4615f80fa..87d9b834a5ae 100644
--- a/third_party/blink/renderer/platform/graphics/canvas_resource_provider.cc
+++ b/third_party/blink/renderer/platform/graphics/canvas_resource_provider.cc
@@ -719,7 +719,7 @@ std::unique_ptr<CanvasResourceProvider> CanvasResourceProvider::Create(
 
     if (size.Width() > max_texture_size || size.Height() > max_texture_size)
       usage = ResourceUsage::kSoftwareResourceUsage;
-
+/*
     is_gpu_memory_buffer_image_allowed =
         (presentation_mode & kAllowImageChromiumPresentationMode) &&
         Platform::Current()->GetGpuMemoryBufferManager() &&
@@ -727,7 +727,7 @@ std::unique_ptr<CanvasResourceProvider> CanvasResourceProvider::Create(
             gfx::Size(size), color_params.GetBufferFormat()) &&
         gpu::IsImageFromGpuMemoryBufferFormatSupported(
             color_params.GetBufferFormat(), context_capabilities);
-
+*/
     is_swap_chain_allowed =
         (presentation_mode & kAllowSwapChainPresentationMode) &&
         context_capabilities.shared_image_swap_chain;
diff --git a/ui/gfx/linux/client_native_pixmap_factory_dmabuf.cc b/ui/gfx/linux/client_native_pixmap_factory_dmabuf.cc
index d8d6e587a847..72adb34f0d7f 100644
--- a/ui/gfx/linux/client_native_pixmap_factory_dmabuf.cc
+++ b/ui/gfx/linux/client_native_pixmap_factory_dmabuf.cc
@@ -60,8 +60,10 @@ class ClientNativePixmapFactoryDmabuf : public ClientNativePixmapFactory {
       case gfx::BufferUsage::SCANOUT_CAMERA_READ_WRITE:
       case gfx::BufferUsage::CAMERA_AND_CPU_READ_WRITE:
       case gfx::BufferUsage::SCANOUT_VEA_READ_CAMERA_AND_CPU_READ_WRITE:
+      /*
         return ClientNativePixmapDmaBuf::ImportFromDmabuf(std::move(handle),
                                                           size, format);
+        */
       case gfx::BufferUsage::GPU_READ:
       case gfx::BufferUsage::SCANOUT:
       case gfx::BufferUsage::SCANOUT_VDA_WRITE:
