diff --git a/third_party/blink/renderer/platform/graphics/canvas_resource_provider.cc b/third_party/blink/renderer/platform/graphics/canvas_resource_provider.cc
index 9c150bee456b..49a2503166ef 100644
--- a/third_party/blink/renderer/platform/graphics/canvas_resource_provider.cc
+++ b/third_party/blink/renderer/platform/graphics/canvas_resource_provider.cc
@@ -737,6 +737,10 @@ std::unique_ptr<CanvasResourceProvider> CanvasResourceProvider::Create(
       GetResourceTypeFallbackList(usage);
 
   const bool is_gpu_memory_buffer_image_allowed =
+  //---***FYDEOS BEGIN***---
+  false;
+  /*
+  //---***FYDEOS END***---
       SharedGpuContext::IsGpuCompositingEnabled() && context_provider_wrapper &&
       presentation_mode == kAllowImageChromiumPresentationMode &&
       gpu::IsImageSizeValidForGpuMemoryBufferFormat(
@@ -745,6 +749,9 @@ std::unique_ptr<CanvasResourceProvider> CanvasResourceProvider::Create(
           color_params.GetBufferFormat(),
           context_provider_wrapper->ContextProvider()->GetCapabilities());
 
+//---***FYDEOS BEGIN***---
+  */
+//---***FYDEOS END***---
   for (CanvasResourceType resource_type : fallback_list) {
     // Note: We are deliberately not using std::move() on
     // |context_provider_wrapper| and |resource_dispatcher| to ensure that the
diff --git a/ui/gfx/linux/client_native_pixmap_factory_dmabuf.cc b/ui/gfx/linux/client_native_pixmap_factory_dmabuf.cc
index 83eba318e72a..5a48f0a3062a 100644
--- a/ui/gfx/linux/client_native_pixmap_factory_dmabuf.cc
+++ b/ui/gfx/linux/client_native_pixmap_factory_dmabuf.cc
@@ -58,8 +58,10 @@ class ClientNativePixmapFactoryDmabuf : public ClientNativePixmapFactory {
       case gfx::BufferUsage::GPU_READ_CPU_READ_WRITE:
       case gfx::BufferUsage::SCANOUT_CAMERA_READ_WRITE:
       case gfx::BufferUsage::CAMERA_AND_CPU_READ_WRITE:
+      /*
         return ClientNativePixmapDmaBuf::ImportFromDmabuf(std::move(handle),
                                                           size);
+      */
       case gfx::BufferUsage::GPU_READ:
       case gfx::BufferUsage::SCANOUT:
       case gfx::BufferUsage::SCANOUT_VDA_WRITE:
