diff --git a/src/mesa/main/fbobject.c b/src/mesa/main/fbobject.c
index 92375a9361b..18cdb247ba4 100644
--- a/src/mesa/main/fbobject.c
+++ b/src/mesa/main/fbobject.c
@@ -234,7 +234,8 @@ get_framebuffer_target(struct gl_context *ctx, GLenum target)
    bool have_fb_blit = _mesa_is_gles3(ctx) || _mesa_is_desktop_gl(ctx);
    switch (target) {
    case GL_DRAW_FRAMEBUFFER:
-      return have_fb_blit ? ctx->DrawBuffer : NULL;
+      //return have_fb_blit ? ctx->DrawBuffer : NULL;
+      return ctx->DrawBuffer;
    case GL_READ_FRAMEBUFFER:
       return have_fb_blit ? ctx->ReadBuffer : NULL;
    case GL_FRAMEBUFFER_EXT:
