diff --git a/src/gallium/drivers/svga/svga_format.c b/src/gallium/drivers/svga/svga_format.c
index 3f68f0cd..c8841aa7 100644
--- a/src/gallium/drivers/svga/svga_format.c
+++ b/src/gallium/drivers/svga/svga_format.c
@@ -268,6 +268,8 @@ svga_translate_screen_target_format_vgpu10(enum pipe_format format)
    switch (format) {
    case PIPE_FORMAT_B8G8R8A8_UNORM:
       return SVGA3D_B8G8R8A8_UNORM;
+   case PIPE_FORMAT_R8G8B8A8_UNORM:
+      return SVGA3D_R8G8B8A8_UNORM;
    case PIPE_FORMAT_B8G8R8X8_UNORM:
       return SVGA3D_B8G8R8X8_UNORM;
    case PIPE_FORMAT_B5G6R5_UNORM:
@@ -2156,6 +2158,7 @@ svga_is_format_supported(struct pipe_screen *screen,
       /* VGPU10 formats */
       case SVGA3D_B8G8R8A8_UNORM:
       case SVGA3D_B8G8R8X8_UNORM:
+      case SVGA3D_R8G8B8A8_UNORM:
       case SVGA3D_B5G6R5_UNORM:
       case SVGA3D_B8G8R8X8_UNORM_SRGB:
       case SVGA3D_B8G8R8A8_UNORM_SRGB:
